less/components/footer.less

```diff
@@ -10,159 +10,130 @@
  */
 
 footer {
-	font-family: @font-family-secondary;
-}
+ font-family: @font-family-secondary;
```

Comment:
Some white-space issues in this file with an extra space added.

Reply:
I noticed this this morning -- might be an issue with my Atom; I usually use two spaces. Will reindent in a separate commit.